java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -323,4 +327,16 @@ public CompletableFuture<Long> scard(String key) {
     public CompletableFuture<Long> exists(@NonNull String[] keys) {
         return commandManager.submitNewCommand(Exists, keys, this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<String> configRewrite() {
+        return commandManager.submitNewCommand(
+                ConfigRewrite, new String[0], this::handleStringResponse);
+    }
+
+    @Override
+    public CompletableFuture<String> configResetStat() {
```

Comment:
same for `configResetStat`.  Without Route, it's only available in the Standalone client.